WooCommerce/Classes/ViewRelated/Orders/Order Details/Receipts/ReceiptViewController.swift

```diff
@@ -0,0 +1,72 @@
+import WebKit
+
+/// Previews a backend-generated receipt
+final class ReceiptViewController: UIViewController {
+    @IBOutlet private weak var webView: WKWebView!
+    private let viewModel: ReceiptViewModel
+
+    init(viewModel: ReceiptViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: "LegacyReceiptViewController", bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        configureContent()
+        configureNavigation()
+    }
+
+    private func configureContent() {
+        guard let receipt = viewModel.receiptRequest else {
+            DDLogError("No receipt could be found.")
```

Comment:
Perhaps log an order number here as well? Or anything else we might need to debug the API after an HE gets this in a log?

Reply:
Good point, I haven't dealt with specific error handling yet, but would be good to leave proper logs for the time being. Updated here: https://github.com/woocommerce/woocommerce-ios/pull/11858/commits/df4cdcf81a307adff924b27042956148c62a08c2